app/webserver/messages.go

```diff
@@ -21,18 +21,23 @@ type ContactListEnvelope struct {
 type DeviceListEnvelope struct {
 	DeviceList []textsecure.DeviceInfo
 }
+type Group struct {
```

Comment:
This all looks really good to me. Only question is, why to you use a new type instead of recycling `store.GroupRecord` ? If it's because of the database field type i would prefer if the type still resides in `store`.

Reply:
I prefer separating definitions for different tasks. GroupRecord is the database representation, not the API representation.
Using GroupRecord, I would have to change the JavaScript code, because the Members are one String instead of a String list in GroupRecord.

Its not sensible to adjust JavaScript code, because of database restrains.